scripts/datasets/VID.py

```diff
@@ -0,0 +1,270 @@
+import json
+import xml.etree.ElementTree as ET
+from os.path import join, isdir
+from os import listdir, mkdir, makedirs
+import glob
+import xml.etree.ElementTree as ET
+from concurrent import futures
+import time
+import os
+import argparse
+import numpy as np
+from gluoncv.utils import download, makedirs
+from gluoncv.utils.filesystem import try_import_cv2
+import tarfile
+import sys
+cv2 = try_import_cv2()
+
+def parse_args():
+    """VID dataset parameter."""
+    parser = argparse.ArgumentParser(
+        description='Download VID dataset and prepare for tracking')
+    parser.add_argument('--download-dir', type=str, default='~/.mxnet/datasets/VID/',
+                        help='dataset directory on disk')
+    parser.add_argument('--instanc-size', type=int, help='instanc image size')
+    parser.add_argument('--num-threads', type=int, help='threads number')
+    args = parser.parse_args()
+    args.download_dir = os.path.expanduser(args.download_dir)
+    return args
+
+args = parse_args()
```

Comment:
There is a `args = parse_args()` in line 266 in main function, please remove this line. 

It is a good practice to write `args` in main function, and pass `args` to other methods instead of using a global `args` variable.